docs/installation/index.md

```diff
@@ -1,16 +1,16 @@
 # Installation
 
-For demonstration, development or testing purposes, RDMO can be installed on Linux, Windows and macOS. If you, however, to set up a production environment, serving RDMO over a Network or the Internet, we suggest that you use a recent Linux distribution, namely CentOS, Debian, or Ubuntu LTS.
+For demonstration, development or testing purposes, RDMO can be installed on Linux, Windows and macOS. If you, however, to set up a production environment, serving RDMO over a Network or the Internet, we suggest that you use a recent Linux distribution. We suggest to use Debian or Ubuntu LTS.
 
-The code is mainly written in Python and should work with a Python higher than 3.6.
+The code is mainly written in Python and works with a version higher than **Python 3.8**.
```

Comment:
When you say higher than in prose it sounds like
Python > 3.8 (not inclusive).

I would recommend:

The code is mainly written in Python and works with version **Python 3.8** or higher.